docs/rtk-query/usage/customizing-queries.mdx

```diff
@@ -390,6 +390,80 @@ const baseQueryWithReauth: BaseQueryFn<
 }
 ```
 
+#### Preventing multiple unauthorized errors
+
+Implementing [`async-mutex`](https://github.com/DirtyHairy/async-mutex) to prevent multiple calls to '/refreshToken' when multiple calls fail with [`401 Unauthorized`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401) errors.
```

Comment:
```suggestion
Using [`async-mutex`](https://github.com/DirtyHairy/async-mutex) to prevent multiple calls to '/refreshToken' when multiple calls fail with [`401 Unauthorized`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401) errors.
```